server/protocols/webadmin-cli/README.md

```diff
@@ -0,0 +1,128 @@
+# A WebAdmin based CLI for Apache James
+
+## Development
+
+Webadmin command-line interface is an upcoming replacement for the outdated, security-vulnerable JMX command-line interface. It also aims at providing a more modern and intuitive interface.
+
+## Run the script
+
+## Syntax
+
+General syntax to run the script
+
+```   
+$ ./james_cli [OPTION] ENTITY ACTION {ARGUMENT}
```

Comment:
If we are doing java, I guess it's gonna be more in the form of `java -jar james_cli.jar [...]` right? :)

Reply:
same remark for other similar syntaxes below

Reply:
I refrained from that comment as the documentation will be corrected later on, once we get something running.

 I'm more interested by the arguments of the command.

Reply:
I changed it.